gslab_scons/misc.py

```diff
@@ -11,28 +11,32 @@
 import _exception_classes
 from size_warning import issue_size_warnings
 
-def scons_debrief(target, env):
-    '''Execute functions after SCons has built all targets'''
+def scons_debrief(args):
+    '''
+    Execute functions after SCons has built all targets.
+    Current list of functions: 
+        1. print state_of_repo
+        2. issue size_warnings
+    '''
     # Log the state of the repo
-    env['CL_ARG'] = env['MAXIT']
-    maxit = int(command_line_args(env))
-    state_of_repo(maxit)
+    args['CL_ARG'] = args['MAXIT']

```

Comment:
I'm a little bit confused by this. What does this do? Why do we have to call `command_line_args` to access maxit?